pkg/cmd/server/api/types.go

```diff
@@ -325,6 +325,14 @@ type AuditConfig struct {
 	// If this flag is set, audit log will be printed in the logs.
 	// The logs contains, method, user and a requested URL.
 	Enabled bool
+	// All requests coming to the apiserver will be logged to this file.
+	Path string
+	// Maximum number of days to retain old audit log files based on the timestamp encoded in their filename.
+	MaxAge int
```

Comment:
I have to stick to days for now, since the luberjack library supports just days.